xml/operations-troubleshooting-ts_blockstorage.xml

```diff
@@ -299,10 +299,12 @@ ansible-playbook -i hosts/verb_hosts _cinder_post_check.yml</screen>
    <emphasis role="bold">&o_blockstore; volume service is down</emphasis>
   </para>
   <para>
-   The &o_blockstore; volume service could be down if the server hosting the volume
-   service fails. In this case you should follow the documented procedure
-   linked below to start the volume service on another controller node. See
-   <xref linkend="sec.operation.manage-block-storage"/> for details.
+   The &o_blockstore; volume service could be down if the server hosting the
+   volume service fails. (Running the command <command>cinder
```

Comment:
I'm unsure if you can answer this @csymons-suse - but what's the potential reasoning for the "could"?